ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -71,7 +71,7 @@ class RecoveringIndexerSpec
         }
     }
 
-    "work when the stream is stopped" in newLoggingContext { implicit logCtx =>
+    "work when the stream is stopped" in newLoggingContext { implicit loggingContext =>
```

Comment:
See if it makes sense to use `LoggingContext.ForTesting` here and remove a level of nesting.